### DISCUSSION TITLE
Docker: get username in per-user statistics instead of just UID?

### DISCUSSION THREAD

**Discussion by zemmyang** [netdata/netdata#15269] [Q&A]
Bit of a stupid question, I guess. There's probably a way to do this with just docker, right? 

Here's my docker run command:
```
docker run -d --name=netdata \
  -e NETDATA_EXTRA_APK_PACKAGES=gcompat \
  --gpus all\
  -p 19999:19999 \
  -v /some/directory/netdataconfig/netdata:/etc/netdata \
  -v netdatalib:/var/lib/netdata \
  -v netdatacache:/var/cache/netdata \
  -v /etc/passwd:/host/etc/passwd:ro \
  -v /etc/group:/host/etc/group:ro \
  -v /proc:/host/proc:ro \
  -v /sys:/host/sys:ro \
  -v /etc/os-release:/host/etc/os-release:ro \
  --hostname=$HOSTNAME \
  --restart unless-stopped \
  --cap-add SYS_PTRACE \
  --security-opt apparmor=unconfined \
  netdata/netdata:stable
```

(It's running on several nodes, and there will be cases where the same username will have several different user IDs.)

**Comment by ilyam8** [comment]
Hi, @zemmyang. What "per-user statistics" are you referring to?

**Reply by zemmyang** [reply]
im looking to get the same thing i'd see from `top` - username, their command, %CPU of the command, memory usage, etc

**Reply by ilyam8** [reply]
You can find that info only when connected to Cloud/using Cloud UI, check the "Functions" tab.

**Reply by zemmyang** [reply]
found it - the user column just gives me the user ID, not the actual name of the user. 

I can put that number into `id -nu 012345678` and I'd get the user's name, but is there a way to do this in netdata automatically?

**Reply by ilyam8** [reply]
I [created](https://learn.netdata.cloud/docs/installing/docker#create-a-new-netdata-agent-container) a ND docker container and the User column contains the username. 

<details>
<summary>screenshot</summary>

<img width="808" alt="Screenshot 2023-06-28 at 14 44 28" src="https://github.com/netdata/netdata/assets/22274335/5b9b255f-7974-420e-8509-a1f8c783bac2">

</details>


I guess it could be user ids for not local users (e.g. LDAP).

**Reply by zemmyang** [reply]
yup, they all connect to the servers that netdata is running on using `ssh user@something`. i think the term is that they're "domain users"?

is it possible to get that info into nd's interface?
